dev/loadgen/prod-benchmark.yaml

```diff
@@ -0,0 +1,42 @@
+## start with
+##    loadgen benchmark prod-benchmark.yaml
+
+workspaces: 500
+ideImage: eu.gcr.io/gitpod-core-dev/build/ide/code:commit-8c1466008dedabe79d82cbb91931a16f7ce7994c
+repos:
+  - cloneURL: https://github.com/gitpod-io/template-typescript-node/
```

Comment:
I tried running test with this repo only and all the ws failed with error: `{"failed":"cannot initialize workspace: cannot initialize workspace: content initializer failed","deployed":1}`

Do we need prereq before running benchmark?

Reply:
There should not be any prerequisites needed, other than having a correct set of images of course (which we do)